app/crt0/crt0_mini_rv32i.s

```diff
@@ -42,6 +42,7 @@ reset_handler:
   la   x2, _stack_start
 
   jal x1, main  //jump to main
+  nop
```

Comment:
Why need this?

Reply:
היה טסט אחד שהוא לא מהקונבנציונלים שאנחנו מריצים (אני לא זוכר מהו) שלא רץ. עשיתי את אותו הדבר בעבר בקבצי cfg אחרים. יש איזה שהוא פספוס בכתיבה הממש אחרונה אם לא מוסיפים את זה. אפשר להוריד אותו ובעתיד אם אעשה שוב את אותו הטסט אז אדע להגיד לך